composer.json

```diff
@@ -71,7 +72,7 @@
         "symfony/monolog-bundle": "^3.1.0",
         "symfony/polyfill-apcu": "^1.0",
         "symfony/psr-http-message-bridge": "^1.0",
-        "symfony/symfony": "3.3.*",
+        "symfony/symfony": "3.3.14",
```

Comment:
why ?

Reply:
I tested 3.3.16 and there are a probleme with tests :s